packages/core/src/external/mpi/merge-patients.ts

```diff
@@ -0,0 +1,37 @@
+import { PatientDataMPI } from "./patient-incoming-schema";
+import { capture } from "../../util/notifications";
+
+export type MergeProtocol = (
+  patients: PatientDataMPI[],
+  demo: PatientDataMPI,
+  cxid?: string
+) => PatientDataMPI | undefined;
+
+export const mergeWithFirstPatient: MergeProtocol = (patients, demo) => {
+  const chosenOne = patients[0];
```

Comment:
this whole file could be simplified of course by just taking the first patient in the matching patient list. However I think this is valuable as part of establishing the place of `merging` in the MPI process, and for its explicitness with respect to what we are actually doing when we take the first matching patient. We are making a decision about a merge according to a protocol, in this case a "choose first" protocol. But as this develops, maybe a better protocol is "highest matching score" or "most pieces of demographic data". I don't know but I think its important to be thinking about this sort of thing. 

The merge protocol does of course elate to the match protocol. If we are greedily searching in match, then there only will ever be one patient in the patients list to "merge" our search patient into